### ISSUE TITLE
Java port of JSON Schema Faker?

### ISSUE THREAD

**Issue by tedepstein** [json-schema-faker/json-schema-faker#501]
JSON Schema Faker seems really useful, but we need a library we can integrate into a Java solution. There are various ways to bridge between Java and JavaScript. We'll explore these. But a native Java port would be ideal. 

If anyone knows of one, has interest in contributing to an open source project to create this, or would use it if it did exist, please chime in.

**Comment by goatandsheep** [comment]
Are you looking for for testing purposes? Because if so, you should just be able to run it in parallel of your Java server and call it through server calls.

**Comment by tedepstein** [comment]
@goatandsheep , no not just for testing purposes. For use in commercial and open source tools for API design, documentation, and development.  JSON examples have a role to play in all of those areas, and we want to offer various ways to generate and manage high-quality example data.

[DiUS/java-faker](https://github.com/DiUS/java-faker) looks promising, though it's missing the JSON Schema piece.

**Comment by yogeshVU** [comment]
@tedepstein were you able to find any alternatives for this?

**Comment by alturkovic** [comment]
I am also interested in this, does anyone know of such library?

**Comment by pateketrueke** [comment]
Closing as out of scope. A Java port is a separate project and outside the scope of this repository.
